### PR TITLE
upgrade bootstrap-frontend-play-28

### PR DESCRIPTION
**Pull request by PaulHodgson**
```
sbt test
...
[info] ScalaTest
[info] Run completed in 15 seconds, 416 milliseconds.
[info] Total number of tests run: 185
[info] Suites: completed 22, aborted 0
[info] Tests: succeeded 185, failed 0, canceled 0, ignored 0, pending 0
[info] All tests passed.
[info] Passed: Total 185, Failed 0, Errors 0, Passed 185
[success] Total time: 17 s, completed 5 May 2022, 08:06:59
```

```
sbt it:test
...
[info] ScalaTest
[info] Run completed in 28 seconds, 687 milliseconds.
[info] Total number of tests run: 41
[info] Suites: completed 4, aborted 0
[info] Tests: succeeded 41, failed 0, canceled 0, ignored 0, pending 0
[info] All tests passed.
[info] Passed: Total 41, Failed 0, Errors 0, Passed 41
[success] Total time: 30 s, completed 5 May 2022, 08:08:14
```

acceptance test run
```
[info] ScalaTest
[info] Run completed in 6 minutes, 32 seconds.
[info] Total number of tests run: 0
[info] Suites: completed 0, aborted 0
[info] Tests: succeeded 0, failed 0, canceled 0, ignored 0, pending 0
[info] No tests were executed.
[info] Passed: Total 44, Failed 0, Errors 0, Passed 44
[success] Total time: 407 s (06:47), completed 5 May 2022, 07:55:46
```

### CODE REVIEW

File: app/actors/ActorService.scala [hunk 1]
```diff
@@ -33,16 +33,16 @@ trait ActorService {
   val logger: Logger = Logger(this.getClass)
 
   def startPollingActor(vaultId: String, archiveId: String)
```
Comment (PaulHodgson) [inline app/actors/ActorService.scala:35]: No need for the `???` in a trait. IntelliJ was warning.

File: build.sbt [hunk 2]
```diff
@@ -14,9 +14,11 @@ lazy val scoverageSettings = {
   )
 }
 
+val akkaVersion = "2.6.19"
```
Comment (PaulHodgson) [inline build.sbt:17]: `akka` updated due to warnings caused by the `bootstrap-frontend-play-28` upgrade.

File: app/actors/ActorService.scala (hunk 1 above)
Comment (greg-gajda) [reply app/actors/ActorService.scala:35]: :+1:

File: build.sbt [hunk 3]
```diff
@@ -30,11 +32,11 @@ def test(scope: String) = Seq(
...
   "com.vladsch.flexmark" % "flexmark-all" % "0.35.10" % scope,
-  "com.typesafe.akka" %% "akka-testkit" % "2.6.14",
-  "com.typesafe.akka" %% "akka-stream"                % "2.6.14",
-  "com.typesafe.akka" %% "akka-slf4j"                 % "2.6.14",
-  "com.typesafe.akka" %% "akka-actor-typed"           % "2.6.14",
-  "com.typesafe.akka" %% "akka-serialization-jackson" % "2.6.14",
+  "com.typesafe.akka" %% "akka-testkit"               % akkaVersion,
+  "com.typesafe.akka" %% "akka-stream"                % akkaVersion,
+  "com.typesafe.akka" %% "akka-slf4j"                 % akkaVersion,
+  "com.typesafe.akka" %% "akka-actor-typed"           % akkaVersion,
```
Comment (greg-gajda) [inline build.sbt:38]: do we use typed actors at all?

Comment (PaulHodgson) [reply build.sbt:38]: I can have a play and try to remove some of these ...

Comment (PaulHodgson) [reply build.sbt]: In fact we just needed the test kit to be explicitly imported.

**Pull request merged**